src/Core/Content/Media/MediaService.php

```diff
@@ -121,7 +121,7 @@ public function fetchFile(Request $request, ?string $tempFile = null): MediaFile
         }
 
         $contentType = $request->headers->get('content_type');
-        if ($contentType === 'application/json') {
+        if (str_starts_with( $contentType, 'application/json')) {
```

Comment:
`str_starts_with ` will not work on PHP 7.4, https://github.com/shopware/production/blob/6.4/composer.json#L54 this is minium version.

Reply:
was suggested in https://github.com/shopware/platform/issues/2196#issuecomment-979997920 
since polyfill is there, it should be no problem

Reply:
Yep, the symfony polyfill will do the trick